### PR TITLE
Main to next

### PR DESCRIPTION
**Pull request by sonalivdeshpande**
The previous integrate PR merged in next branch: https://github.com/microsoft/FluidFramework/pull/10652 was squash-merged instead of merge. Instead of reverting merged PR, we can merge this PR again to preserve the git history. It has all commit upto June 14th.

Commits revert in this PR: 
https://github.com/sonalivdeshpande/FluidFramework/commit/1c2ec973d55961d892534f20933b97a536e8adff
https://github.com/sonalivdeshpande/FluidFramework/commit/b55722a8d741f8bcae2c2e01675417a7e0300ede

server packages in next should be on 0.1037.1000 and protocol defs on 0.1029.1000

https://github.com/microsoft/FluidFramework/commit/d7ac155fe261e195ce0c5776cd4b8a4685429d51 -- opening a separate PR for this commit

### CODE REVIEW

**Comment by ssimic2** [comment]
Can you please update description, if you are not reverting squashed PR please share explanation so we have some history here. Also please describe individual commits that were reverted and why.

**Review by tylerbutler — APPROVED** [review]

File: azure/packages/azure-client/src/packageVersion.ts [hunk 1]
```diff
@@ -6,4 +6,4 @@
  */
 
 export const pkgName = "@fluidframework/azure-client";
-export const pkgVersion = "0.60.1000";
+export const pkgVersion = "1.1.0";
```
Comment (ssimic2) [inline azure/packages/azure-client/src/packageVersion.ts:9]: @sonalivdeshpande did we accidently bumped  azure client in main?

File: packages/dds/ordered-collection/package.json [hunk 2]
```diff
@@ -75,7 +75,7 @@
     "@fluidframework/build-tools": "^0.2.71273",
     "@fluidframework/eslint-config-fluid": "^0.28.2000",
     "@fluidframework/mocha-test-setup": "^2.0.0",
-    "@fluidframework/ordered-collection-previous": "npm:@fluidframework/ordered-collection@^0.59.0",
+    "@fluidframework/ordered-collection-previous": "npm:@fluidframework/ordered-collection@^1.0.0",
```
Comment (ssimic2) [inline packages/dds/ordered-collection/package.json:78]: We need to sort out auto-integration story here. Every time we do main-to-next integration, we are picking up "previous" from main, that may not be correct.

File: azure/packages/azure-client/src/packageVersion.ts (hunk 1 above)
Comment (sonalivdeshpande) [reply azure/packages/azure-client/src/packageVersion.ts:9]: > Why [d7ac155](https://github.com/microsoft/FluidFramework/commit/d7ac155fe261e195ce0c5776cd4b8a4685429d51)? was reverted ?

This has conflicts and build failures. opening this commit as a separate PR against next

**Review by ssimic2 — APPROVED** [review]

File: azure/packages/azure-client/src/packageVersion.ts (hunk 1 above)
Comment (sonalivdeshpande) [reply azure/packages/azure-client/src/packageVersion.ts:9]: azure client in main is bumped to 1.1.0: https://github.com/microsoft/FluidFramework/pull/10672. I think @tylerbutler would be bumping the azure client in next to 2.0.0 after the release?

**Pull request merged**